ci/main.py

```diff
@@ -1,2 +1,5 @@
 def get_hello():
     return "hello"
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/ci/main.py
+++ b/ci/main.py
@@ -1,5 +1,6 @@
 def get_hello():
     return "hello"
 
+
 def get_world():
     return "world"
```